gazebo_ros/include/gazebo_ros/gazebo_ros_api_plugin.h

```diff
@@ -291,6 +292,9 @@ class GazeboRosApiPlugin : public SystemPlugin
   /// \brief Unused
   void onResponse(ConstResponsePtr &response);
 
+  /// \brief Subscriber callback for performance metrics. This will be send in the ROS network
+  void onPerformanceMetrics(const boost::shared_ptr<gazebo::msgs::PerformanceMetrics const> &msg);
```

Comment:
Could you add some ifdefs so this works with the current minor version of Gazebo 11? We should proceed with the assumption that the Gazebo 11 packages on https://packages.ros.org won't be upgraded, and this needs to work with that.